src/inline-assembly.md

```diff
@@ -184,7 +184,7 @@ Here is the list of currently supported register classes:
 >
 > - On x86-64 the high byte registers (e.g. `ah`) are not available in the `reg_byte` register class.
 >
-> - Some register classes are marked as "Only clobbers" which means that they cannot be used for inputs or outputs, only clobbers of the form `out("reg") _` or `lateout("reg") _`.
```

Comment:
This change is incorrect: clobber-only registers can only be used with explicit registers, not register classes.